main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Drama"]
+    for genre_name in genres:
+        Genre.objects.create(name=genre_name)
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Actor.objects.all()
+    Genre.objects.filter(name="Dramma", ).update(name="Drama", )
```

Comment:
When you pass one argument using any operation, there is no need to place trailing comma and space
```suggestion
    Genre.objects.filter(name="Dramma").update(name="Drama")
```
Apply this change to all places where applicable.